packages/ketcher-polymer-editor-react/src/components/menu/Menu.tsx

```diff
@@ -0,0 +1,110 @@
+/****************************************************************************
+ * Copyright 2021 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***************************************************************************/
+import { MenuItem } from './menuItem'
+import styled from '@emotion/styled'
+import { Divider } from 'components/menu/menuDivider'
+import { MenuItemVariant } from 'components/menu/menu.types'
+
+interface GroupItem {
+  name: MenuItemVariant
+  options?: MenuItemVariant[]
+  vertical?: boolean
+}
+
+interface GroupPropTypes {
+  items: GroupItem[]
+}
+
+const Group = ({ items }: GroupPropTypes) => {
+  const GroupContainer = styled('div')`
+    display: flex;
+    flex-direction: column;
+    flex-wrap: nowrap;
+    align-items: center;
+    background-color: ${(props) => props.theme.color.background.primary};
+    border-radius: 2px;
+
+    > * {
+      margin-bottom: 8px;
+    }
+
+    > :last-child {
+      margin-bottom: 0;
+    }
+  `
+  return (
+    <GroupContainer>
+      {items.map((item: GroupItem) => {
+        return (
+          <MenuItem
+            key={item.name}
+            name={item.name}
+            options={item.options}
+            vertical={item.vertical}
+          />
+        )
+      })}
+    </GroupContainer>
+  )
+}
+
+const MenuComponent = styled('div')`
+  width: 32px;
+  > * {
+    margin-bottom: 8px;
+  }
+`
+
+const Menu = () => {
+  return (
+    <MenuComponent>
+      <Group
+        items={[
+          {
+            name: 'open'
+          }
+        ]}
+      />
+      <Divider />
+      <Group items={[{ name: 'undo' }]} />
+      <Group
+        items={[
+          { name: 'erase' },
+          {
+            name: 'select',
+            options: ['select-lasso', 'select-rectangle', 'select-fragment'],
+            vertical: true
+          },
+          { name: 'shapes', options: ['rectangle', 'ellipse'] },
+          { name: 'redo' }
+        ]}
+      />
+      <Group
```

Comment:
it would be better to  implement it as subcomponents. look at https://dev.to/ms_yogii/create-react-subcomponents-in-a-simple-way-5h1f  for example